client/script.js

```diff
@@ -122,16 +122,12 @@ var payWithOxxo = function(stripe, clientSecret) {
           }
         }
       },
-      { handleActions: false }
+      { handleActions: true }
```

Comment:
We don't need to include this field do we? It defaults to true

Reply:
You're right, BTW I tried both ways :)